libsoftwarecontainer/component-test/filegateway_componenttest.cpp

```diff
@@ -37,27 +38,54 @@ class FileGatewayTest : public SoftwareContainerGatewayTest
 
         gw = std::unique_ptr<FileGateway>(new FileGateway(m_container));
 
-        // Create file
-        ASSERT_TRUE(writeToFile(FILE_PATH, FILE_CONTENT));
+        // Create files
+        ASSERT_EQ(mkdir(DIR_PATH.c_str(),775),0);
+        ASSERT_TRUE(createFile(FILE_PATH, FILE_CONTENT));
```

Comment:
I'm not a fan of testing things in the test setup. Do we think this can fail? It should be a test in that case, a separate test.